### PR TITLE
[WIP] Create README

### LINKED ISSUE #3: [DOC] Write README
## Problem to solve

[//]: # "A clear and concise description of what the problem is"
Project have no README

## Further details

[//]: # "A clear and concise description of what you want to happen."
- Write README to introduce about this project

### PR DESCRIPTION
**Pull request by nvti**
## What does this MR do?

[//]: # "Briefly describe what this MR is about"
Create README
## Related issues

[//]: # "Link related issues below. Insert the issue link or reference after the word `Closes` if merging this should automatically close it."
Close #3 
## Author's checklist

- [x] Link related issues. If no related issue, create issue first.
- [x] Target to right branch
- [x] Set milestone
- [x] Assign to a reviewer
- [x] Apply label.

## Review checklist

- [x] Former merge request
- [x] Right milestone
- [x] Code/Document
  - [x] Follow [coding convention]()
  - [x] Follow [Documentation Guidelines]()
  - [x] Clarity
  - [x] Completeness
- [x] This merge request is not conflict

### CODE REVIEW

**Review by honghanhh — APPROVED** [review]

**Comment by nvti** [comment]
@honghanhh đã viết xong chưa mà approve :))

File: README.md [hunk 1]
```diff
@@ -1 +1,45 @@
...
+    - [ ] Constituency Parsing and Tree Recursive Neural Networks.
+- __Week 10__:
+    - [ ] Recent Advances in Low Resource Machine Translation.
+    - [ ] Future of NLP + Deep Learning.
+- __Week 11__:
+    - [ ] Final project poster session.
+
+### Contributors:
+- [Viet-Tien](https://github.com/tiena2cva)
+- [honghanhh](https://github.com/honghanhh)
```
Comment (nvti) [inline README.md:48]: Need a newline at the end of every file

File: README.md [hunk 2]
```diff
@@ -1 +1,45 @@
-# cs224n_2020
\ No newline at end of file
+## __CS224n: Natural Language Processing with Deep Learning__
```
Comment (nvti) [inline README.md]: Should be header 1

File: README.md [hunk 3]
```diff
@@ -1 +1,45 @@
...
+
+## Course links
+- Course page [CS224N](http://web.stanford.edu/class/cs224n/).
+- Lecture videos 2020: (Updating).
+
+### Table of contents
```
Comment (nvti) [inline README.md]: Should be header 2

File: README.md [hunk 4]
```diff
@@ -1 +1,45 @@
...
+- __Week 9__:
+    - [ ] Fairness and Inclusion in AI.
+    - [ ] Constituency Parsing and Tree Recursive Neural Networks.
+- __Week 10__:
+    - [ ] Recent Advances in Low Resource Machine Translation.
+    - [ ] Future of NLP + Deep Learning.
+- __Week 11__:
+    - [ ] Final project poster session.
+
+### Contributors:
```
Comment (nvti) [inline README.md]: Should be header 2

**Review by nvti — COMMENTED** [review]
Good job, but still has some mistake

File: README.md (hunk 2 above)
Comment (honghanhh) [reply README.md]: Header 1 làm title xuống dòng trông hơi xấu @tiena2cva

File: README.md (hunk 1 above)
Comment (honghanhh) [reply README.md:48]: ok mate :+1:

File: README.md (hunk 2 above)
Comment (nvti) [reply README.md]: English, pls

The title should be header 1. If you want a beautiful title, you need to change the title to shorter

File: README.md [hunk 5]
```diff
@@ -1 +1,49 @@
-# cs224n_2020
\ No newline at end of file
+# __CS224n: Natural Language Processing with Deep Learning__
```
Comment (nvti) [inline README.md:1]:
```suggestion
# CS224n: Natural Language Processing Course
```

File: README.md (hunk 1 above)
Comment (nvti) [reply README.md:48]:
```suggestion
- [honghanhh](https://github.com/honghanhh)

```

File: README.md (hunk 5 above)
Comment (nvti) [reply README.md:1]: No need change title shorter? @honghanhh

**Comment by honghanhh** [comment]
@tiena2cva  is there any problem to approve the request?

**Pull request merged**